.github/workflows/test_pyiron.yml

```diff
@@ -5,33 +5,31 @@ name: Test case pyiron script jobs
 on: [push]
 
 jobs:
-  build-linux:
-    runs-on: ubuntu-latest
+  test_pyiron:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ubuntu-latest]
 
     steps:
-    - name: Checkout
-      uses: actions/checkout@v2
+    - uses: actions/checkout@v2
+    - uses: conda-incubator/setup-miniconda@v2
       with:
-        submodules: recursive
-    - name: Set up Python 3.9
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.9
-    - name: Install conda environment
-      run: |
-        $CONDA/bin/conda env update --file ./environment.yml --name base
-    - name: cp template files
+        python-version: "3.9"
+        mamba-version: "*"
+        channels: conda-forge
+        channel-priority: strict
+        auto-update-conda: true  
+        environment-file: environment.yml
+    - name: adding pyiron config file and pyiron resources dir
+      shell: bash -l {0}
       run: |
-        DIR_TEMPLATES=$CONDA/share/pyiron/templates
-        echo $DIR_TEMPLATES
-        DIR_TENSILETEST=$DIR_TEMPLATES/tensiletest
-        echo $DIR_TEMPLATES
-        echo $GITHUB_WORKSPACE
-        mkdir -p $DIR_TENSILETEST
-        cp $GITHUB_WORKSPACE/script.py $DIR_TENSILETEST/.
-        cp $GITHUB_WORKSPACE/input.json $DIR_TENSILETEST/.
-        echo $DIR_TENSILETEST
-        ls -lh $DIR_TENSILETEST
+        echo $CONDA
+        cp -r .ci_support/pyiron ./
```

Comment:
This works in the CI, but I would prefer having a similar script that creates the corresponding environment on the local machine (previously make_environment.sh which has been removed). So this config file would have to be generated on a local install as well using the above command. Wouldn't it make sense (in order to avoid duplicating the code) adding that into a separate bash script (or something more OS agnostic)  that can be called both on the local level as well as in the CI?

Reply:
In the local setup, users need to have a `.pyiron` file, in which they define a path for `pyiron_resource` folder. This path should include a templates folder, and the TensileTest folder should be copied there. I will try to clarify it in a README, but I can provide a bash or python script for this purpose as well.